config/application.yml.example

```diff
@@ -188,6 +188,7 @@ test:
   logins_per_ip_limit: '2'
   logins_per_ip_period: '60'
   max_mail_events: '2'
+  mandrill_api_token: '123abc'
```

Comment:
Are you familiar with our process for adding new configs like this? We'll need a corresponding devops PR.